src/VersionManager.cs

```diff
@@ -19,68 +20,166 @@ internal class VersionManager
         /// <summary>
         /// The latest version of DarkRift.
         /// </summary>
-        private static string latestDarkRiftVersion;
+        private static Version latestDarkRiftVersion;
 
         /// <summary>
         /// Gets the path to a specified installation, downloading it if required.
         /// </summary>
         /// <param name="version">The version number required.</param>
         /// <param name="pro">Whether the pro version should be used.</param>
-        /// <param name="netStandard">Whether the .NET Standard build should be used.</param>
+        /// <param name="platform">Whether the .NET Standard build should be used.</param>
         /// <returns>The path to the installation, or null, if it cannot be provided.</returns>
-        public static string GetInstallationPath(string version, ServerTier tier, ServerPlatform platform)
+        public static string GetInstallationPath(Version version, ServerTier tier, ServerPlatform platform)
         {
-            string fullPath = Path.Combine(USER_DR_DIR, "installed", tier.ToString().ToLower(), platform.ToString().ToLower(), version);
+            return Path.Combine(USER_DR_DIR, "installed", tier.ToString().ToLower(), platform.ToString().ToLower(), version.ToString());
```

Comment:
I feel like this should throw an `InvalidOperationException` if the version isn't actually installed. It seems strange it can return a path that doesn't actually exist.